Task-6/Task-6/Services/Messages.cs

```diff
@@ -0,0 +1,67 @@
+using NLog;
+using Task_6.Models;
+
+namespace Task_6.Services;
+
+public class Messages
+{
+    public static Logger _logger = LogManager.GetCurrentClassLogger();
+
```

Comment:
Here add method Info(string message), and at all places where you write logger.Info() write just Info()